meta-facebook/yv35-cl/src/platform/plat_sensor_table.c

```diff
@@ -425,6 +425,58 @@ void check_vr_type(uint8_t index)
 	}
 }
 
+void check_outlet_temp_type(uint8_t index)
```

Comment:
Add check to make sure index is within bounds

Reply:
I already add index check. Please review it again.